transformer_engine/pytorch/module/grouped_linear.py

```diff
@@ -617,20 +658,26 @@ def forward(
             grad_output_quantizers, _ = [None] * self.num_gemms, [None] * self.num_gemms
             if self.fp8:
                 input_quantizers = [
-                    self.quantizers["scaling_fwd"][self._offsets["input"] + i]
+                    self.quantizers["scaling_fwd"][
```

Comment:
This is a breaking change for delayed scaling. The checkpoint saved with older TE versions loaded with new TE versions could be wrong.

But it's necessary for blockwise scaling.

Should we add a switch for delayed scaling and others? cc @timmoon10

Reply:
I think we broke checkpoints in the 2.0 release, so it's a somewhat narrow window where this change would affect customers. This is fine with me.